### PR TITLE
[BRAN-558] Automate Mosaic Testing Workflow

### PR DESCRIPTION
**Pull request by Alderr**
## Purpose/Description:

 - Upgraded Storybook to 8
 - Added npm commands to build with rollup and use yalc autonomously

## Jira Link:

[BRAN-558](https://collagegroup.atlassian.net/browse/BRAN-558?atlOrigin=eyJpIjoiOGNiMDRhNTNlNmNiNDEyYzk0MWE4NTJmZmQ2MDhkYzAiLCJwIjoiaiJ9)

[BRAN-558]: https://collagegroup.atlassian.net/browse/BRAN-558?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/colors/Colors.mdx [hunk 1]
```diff
@@ -15,10 +15,23 @@ import {
 
 <ColorPalette>
   <ColorItem title="Collage Red" colors={collage.red} />
-  <ColorItem title="Multicultural Yellow" colors={collage.yellow} />
-  <ColorItem title="Generations Orange" colors={collage.orange} />
-  <ColorItem title="LGBTQ+ Purple" colors={collage.purple} />
-  <ColorItem title="Gender Teal" colors={collage.teal} />
+
+{' '}
```
Comment (diego20) [inline src/colors/Colors.mdx]: Why these spaces?

File: src/components/Template.mdx [hunk 2]
```diff
@@ -1,10 +1,10 @@
-import { Meta } from '@storybook/addon-docs';
+import { Meta } from '@storybook/blocks';
 
 <Meta title="Guide/Component Story MDX Template" />
 
 # Component Name Here
 
-**DATE ADDED**: [Date]
+**DATE ADDED**: \[Date]
```
Comment (diego20) [inline src/components/Template.mdx]: Is this `\` needed or a Typo?

**Review by diego20 — APPROVED** [review]
Can you please update the doc you created about pubilshing the packages with these new NPM commands?

File: src/components/Template.mdx (hunk 2 above)
Comment (Alderr) [reply src/components/Template.mdx]: @Alderr i think Prettier/lint/storybook migration added things for no reason

File: src/colors/Colors.mdx (hunk 1 above)
Comment (Alderr) [reply src/colors/Colors.mdx]: @Alderr i think Prettier/lint/storybook migration added things for no reason

**Comment by Alderr** [comment]
@diego20 Publishing for mosaic didn't change, only how we tested changes.

**Comment by diego20** [comment]
> @diego20 Publishing for mosaic didn't change, only how we tested changes.

Right, I meant to document that. There are like 5 new NPM commands. Would be nice to have a guide on what they do and how to test now.

**Comment by Alderr** [comment]
@diego20 It is updated, heres the [link.](https://collagegroup.atlassian.net/wiki/spaces/SD/pages/92536834/Mosaic+101#Testing-Your-Component)

Some of the new commands are also one big command split so it's easier to read & tweak for the future.

**Review by TrevorCollins — APPROVED** [review]
This is so exciting! I never got the old workflow to work for me. But this was so much easier. Thank you! Looks good

**Pull request merged**